src/tagmanager/tm_ctags.c

```diff
@@ -478,14 +489,26 @@ const gchar *tm_ctags_get_lang_kinds(TMParserType lang)
 
 const gchar *tm_ctags_get_kind_name(gchar kind, TMParserType lang)
 {
-	kindDefinition *def = getLanguageKindForLetter(lang, kind);
+	kindDefinition *def;
+
+	if (lang == TM_PARSER_NONE)
+		return "unknown";
+
+	def = getLanguageKindForLetter(lang, kind);
```

Comment:
Maybe something like this to avoid  repeating the fallback value?
```suggestion
	kindDefinition *def = NULL;

	if (lang != TM_PARSER_NONE)
		def = getLanguageKindForLetter(lang, kind);
```

Reply:
Yes, that's better - done at both places.